resources/tests.mjs

```diff
@@ -104,8 +104,8 @@ Suites.push({
 });
 
 Suites.push({
-    name: "React-TodoMVC",
-    url: "todomvc/architecture-examples/react/index.html",
+    name: "TodoMVC-React",
+    url: "todomvc/architecture-examples/react/dist/index.html",
     async prepare(page) {
         const element = await page.waitForElement(".new-todo");
         element.focus();
```

Comment:
This is a comment for the line `dispatchEvent("input")` below, and this can be handled in a separate pull request.

Current versions of react (since ~v16) have some code that tracks the `value`. As a result the `input` event doesn't do anything now:
* when `setValue` is called, react saves the `value`
* when the event is dispatched, react compares the current `value` with the saved one, they are the same value => as a result it swallows the `input` event. See https://github.com/facebook/react/issues/10135 for more discussion about this.

The solution that the `testing-library` library uses is this `setNativeValue` function that we might want to steal to replace our current implementation of `setValue`:
https://github.com/testing-library/dom-testing-library/blob/fb069c93983bc0300a6e1c91bdec5bf9443b5286/src/events.js#L106-L123

This has no impact to _this_ version of todomvc because this version doesn't use the `input` or `change` event (as opposed to the previous one), that's why it can be handled separely.